### DISCUSSION TITLE
no gui option with Octave 8.3.0_1 install

### DISCUSSION THREAD

**Discussion by mdb5** [Homebrew/discussions#4762] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.7-43-ge7d0625
ORIGIN: https://github.com/Homebrew/brew
HEAD: e7d0625776dca58c81b03c19eeda1af8bd6bb58e
Last commit: 13 hours ago
Core tap JSON: 03 Sep 03:26 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.kLADdI0QZv/org.xquartz:0
HOMEBREW_MAKE_JOBS: 16
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 16-core 64-bit kabylake
Clang: 14.0.3 build 1403
Git: 2.42.0 => /usr/local/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.5.1-x86_64
CLT: 14.3.1.0.1.1683849156
Xcode: 14.3.1

~
```

### Output of `brew doctor`

```shell
Your system is ready to brew.

~
```

### Description of issue

Upgraded Octave from 8.3.0 to 8.3.0_1 and the octave --gui option fails  GUI version had been working fine.  Cannot figure out how to rewind to previous version.

output:   octave: GUI features missing or disabled in this build

______
_So, I tried installing from source..._

I get these errors/warnings listed in the verbose output of the brew build script (build from source and using gcc-13)...

> checking Qt version 5... checking for Qt5Core Qt5Gui Qt5Help Qt5Network Qt5PrintSupport Qt5Xml... no
> configure: WARNING: Qt libraries not found; disabling Qt GUI

I suspect this is flagging a key problem.

In my case, Qt@5 is installed in the cellar.  The needed folder names are found in this folder:
          /usr/local/Cellar/qt@5/5.15.10_1/lib/cmake

How do I get the install script to find these files?

**Comment by iMichka** [comment, marked as answer]
https://github.com/Homebrew/homebrew-core/pull/141309 will fix this. Sorry for the breakage.

**Reply by mdb5** [reply]
version 8.3.0_2 resolved the problem.

**Reply by tangjinchuan** [reply]
Hi, the same thing happened now with octave 9.1 on an M1 MacBook with OS 14.4.
